### DISCUSSION TITLE
Trame course coming up (Feb 12th)

### DISCUSSION THREAD

**Discussion by jourdain** [Kitware/trame#410] [Announcements]
Kitware will be hosting a remote trame training course on February 12th from 1pm - 5pm EST. Be sure to register for the course using the code TRAMECOURSE for a 20% off registration discount!

For more information, see: https://www.kitware.com/courses/trame/

**Comment by Jimmy-KL** [comment]
Will the course be recorded, so that I can visit it again?

**Reply by jourdain** [reply]
Usually we don't record the paid courses unless it is for a single company. Are you planning to attend?

**Reply by Jimmy-KL** [reply]
Yes, I plan to attend. I asked whether it would be recorded because the time is not convenient for me. It's midnight in my time zone. Whatever, I will try to manage it.

**Reply by jourdain** [reply]
Kitware France is planning a course on trame soon which might be easier for your time zone?

**Reply by Jimmy-KL** [reply]
Yes, France can be easier. Do you know when it will start?

**Reply by finetjul** [reply]
It will be on March 14th, and start at 9AM (CET /GMT+1).

**Reply by Jimmy-KL** [reply]
@jourdain I registered for the course on February 12th. The online course link will be sent to my registered email before the course, right?

**Reply by jourdain** [reply]
yes it will. Thanks for registering.
